src/api/utilities.ts

```diff
@@ -0,0 +1,95 @@
+import { connect, Utils } from '..';
+import { Client } from '../client';
+
+let saveClient: (clientData: string | null) => void;
+let loadClient: () => Client | undefined;
```

Comment:
Not to overcomplicate things but I feel like these functions should go in a file called `state.ts` because "utilities" is generally used for stateless functions

Reply:
Added in [235fd9e](https://github.com/GridPlus/gridplus-sdk/pull/525/commits/235fd9e7f44a6044fb7e5e6f850b64af23c5b4fc)